modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommercePriceOnApplication.testcase

```diff
@@ -572,6 +639,362 @@ definition {
+
+			CommerceSKUs.setSkuPricing(priceOnApplication = "true");
+
+			Button.clickSave();
+		}
+
+		task ("And the admin sets up the account eligibility for Commerce Account") {
+			CommercePriceLists.openPriceListsAdmin();
+
+			CommerceNavigator.gotoEntry(entryName = "Test Price List 1");
+
+			CommerceEntry.gotoMenuTab(menuTab = "Eligibility");
+
+			CommerceEntry.setEntryEligibility(accountsEligibility = "Commerce Account");
+		}
+
+		task ("When the buyer for Commerce Account navigates to the Minium Catalog page") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+		}
+
+		task ("Then the product is marked as Price on Application") {
+			AssertElementPresent(
+				key_productName = "U-Joint",
+				locator1 = "CommerceAccelerators#PRODUCT_CARD_PRICE_ON_APPLICATION");
+		}
+
+		task ("But if buyer2 for Commerce Account 2 navigates to the Minium Catalog page") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer2@liferay.com");
+		}
+
+		task ("Then the product is not marked as Price on Application") {
+			AssertElementNotPresent(
+				key_productName = "U-Joint",
+				locator1 = "CommerceAccelerators#PRODUCT_CARD_PRICE_ON_APPLICATION");
+		}
+
+		task ("When the admin sets up the account group eligibility for Account Group Name that contains Commerce Account 2") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+
+			CommercePriceLists.openPriceListsAdmin();
+
+			CommerceNavigator.gotoEntry(entryName = "Test Price List 1");
+
+			CommerceEntry.gotoMenuTab(menuTab = "Eligibility");
+
+			CommerceEntry.setEntryEligibility(accountGroupsEligibility = "Account Group Name");
+		}
+
+		task ("And buyer2 for Commerce Account 2 navigates to the Minium Catalog page") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer2@liferay.com");
+		}
+
+		task ("Then the product is marked as Price on Application") {
+			AssertElementPresent(
+				key_productName = "U-Joint",
+				locator1 = "CommerceAccelerators#PRODUCT_CARD_PRICE_ON_APPLICATION");
+		}
+
+		task ("But if the buyer for Commerce Account navigates to the Minium Catalog page") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+		}
+
+		task ("Then the product is not marked as Price on Application") {
+			AssertElementNotPresent(
+				key_productName = "U-Joint",
+				locator1 = "CommerceAccelerators#PRODUCT_CARD_PRICE_ON_APPLICATION");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-11685. Buyer can use Price on Application with eligible channel"
+	@priority = 5
+	test CanUsePriceOnApplicationChannelEligibility {
```

Comment:
Can you change the name in: "CanPriceListUsePriceOnApplicationChannelEligibility"
Thanks!